src/api/build.gradle

```diff
@@ -99,3 +100,7 @@ graalvmNative {
         }
     }
 }
+
+dockerfileNative {
+    instruction "RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.11/main' >> /etc/apk/repositories && apk update && apk add 'zlib<1.2.12'"
+}
```

Comment:
This is a temporary solution for https://github.com/madler/zlib/issues/613 and  https://github.com/madler/zlib/issues/634 . Once zlib 1.2.13 is included in stable alpine linux releases we can remove this.